x-pack/plugins/ml/public/jobs/new_job/simple/single_metric/create_job/create_job_service.js

```diff
@@ -214,7 +214,7 @@ export function SingleMetricJobServiceProvider() {
             buckets: {
               date_histogram: {
                 field: formConfig.timeField,
-                interval: interval
+                fixed_interval: interval
```

Comment:
Will this include units or just be a bare number?  I just realised not specifying any units is also deprecated in 7.2, so if there are currently no units it should append `"ms"`.

Same in the other places in this file.

Reply:
fixed in 105d2ea65be9531ad1d2739680f4cbd1a3e55b37